windows/libdatadog.props

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003"
+        ToolsVersion="15.0">
+  <PropertyGroup>
+    <LIBDDPROF-PLATFORM Condition="'$(Platform)'=='x64'">x64</LIBDDPROF-PLATFORM>
+    <LIBDDPROF-PLATFORM Condition="'$(Platform)'=='Win32' OR '$(Platform)'=='x86'">x86</LIBDDPROF-PLATFORM>
```

Comment:
Minor: Should this be `libdatadog` instead of `libddprof`?

Reply:
oh good point.

Yeah the risk is low, since it used to build `nuget package`. 
I tested the generated package on the profiler and it looks like it's working.
This has been migrated from a `nuspec` file (old way to describe nuget package generation)